docker-compose.test.yml

```diff
@@ -20,6 +20,7 @@ services:
       - FHIR_PAYLOAD_DIR=../bcdaworker/data
       - FHIR_STAGING_DIR=../bcdaworker/tmpdata
       - FHIR_ARCHIVE_DIR=../bcdaworker/archive
+      - PENDING_DELETION_DIR=./tempCCLFDir
```

Comment:
Is this line relevant to this PR?

Reply:
I added this after Deirdre pointed out the "invalid cross-device link" errors when the `import-synthetic-cclf-package` CLI command is used. Since it's not related to this ticket, I'll remove it and create a ticket to address this separately.